### PR TITLE
Update Slack and mailing list links, add CNCF logo

### PR DESCRIPTION
**Pull request by scholzj**
This PR updates the links to Slack and mailing lists to pooint to the new CNCF hosted maling lists and Slack. It also adds a CNCF logo into the README file.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -19,14 +19,13 @@ The [documentation contributor guide][doc-contrib-guide] describes how to contri
 
 If you want to get in touch with us first before contributing, you can use:
 
-* [Strimzi mailing list][mailinglist]
-* [Strimzi Slack workspace][slack]
+- [#strimzi channel on CNCF Slack](https://slack.cncf.io/)
```
Comment (ppatierno) [inline CONTRIBUTING.md]: on the sandbox projects page they are using this link, maybe it's somehow direct to the channel? 

https://cloud-native.slack.com/messages/strimzi

File: README.md [hunk 2]
```diff
@@ -23,8 +23,9 @@ Documentation to the current _master_ branch as well as all releases can be foun
 
 If you encounter any issues while using Strimzi, you can get help using:
 
-- [Strimzi mailing list](https://www.redhat.com/mailman/listinfo/strimzi)
-- [Strimzi Slack workspace](https://join.slack.com/t/strimzi/shared_invite/enQtMzU2Mjk3NTgxMzE5LTYyMTUwMGNlMDQwMzBhOGI4YmY4MjhiMDgyNjA5OTk2MTFiYjc4M2Q3NGU1YTFjOWRiMzM2NGMwNDUwMjBlNDY)
+- [#strimzi channel on CNCF Slack](https://slack.cncf.io/)
```
Comment (ppatierno) [inline README.md]: ditto as above.

File: README.md [hunk 3]
```diff
@@ -45,12 +46,19 @@ The [Documentation Contributor Guide](https://strimzi.io/contributing/guide/) de
 
 If you want to get in touch with us first before contributing, you can use:
 
-- [Strimzi mailing list](https://www.redhat.com/mailman/listinfo/strimzi)
-- [Strimzi Slack workspace](https://join.slack.com/t/strimzi/shared_invite/enQtMzU2Mjk3NTgxMzE5LTYyMTUwMGNlMDQwMzBhOGI4YmY4MjhiMDgyNjA5OTk2MTFiYjc4M2Q3NGU1YTFjOWRiMzM2NGMwNDUwMjBlNDY)
+- [#strimzi channel on CNCF Slack](https://slack.cncf.io/)
```
Comment (ppatierno) [inline README.md]: ditto as above

File: CONTRIBUTING.md (hunk 1 above)
Comment (scholzj) [reply CONTRIBUTING.md]: That links is useless unless you are already registered. So I think we should keep what I have there.

File: README.md (hunk 2 above)
Comment (scholzj) [reply README.md]: That links is useless unless you are already registered. So I think we should keep what I have there.

File: README.md (hunk 3 above)
Comment (scholzj) [reply README.md]: That links is useless unless you are already registered. So I think we should keep what I have there.

File: README.md [hunk 4]
```diff
@@ -23,8 +23,9 @@ Documentation to the current _master_ branch as well as all releases can be foun
 
 If you encounter any issues while using Strimzi, you can get help using:
 
-- [Strimzi mailing list](https://www.redhat.com/mailman/listinfo/strimzi)
-- [Strimzi Slack workspace](https://join.slack.com/t/strimzi/shared_invite/enQtMzU2Mjk3NTgxMzE5LTYyMTUwMGNlMDQwMzBhOGI4YmY4MjhiMDgyNjA5OTk2MTFiYjc4M2Q3NGU1YTFjOWRiMzM2NGMwNDUwMjBlNDY)
+- [#strimzi channel on CNCF Slack](https://slack.cncf.io/)
+- [Strimzi Users mailing list](https://lists.cncf.io/g/cncf-strimzi-users/topics)
+- [Strimzi Dev mailing list](https://lists.cncf.io/g/cncf-strimzi-dev/topics)
```
Comment (tombentley) [inline README.md]: Don't we expect people with problems using Strimzi to use the User's mailing list, and people wanting to develop Strimzi to use the dev mailing list?

Comment (scholzj) [reply README.md]: Should be fixed now.

**Pull request merged**